GOVERNANCE.md

```diff
@@ -0,0 +1,205 @@
+
+The Eiffel community proactively adopts and follows best practices employed by successful open source communities while maintaining a governance agreeable to it's members.
+
+# Governance
+
+Eiffel is a community of projects working on building CI/CD Pipelines using Eiffel Protocol.
+
+The projects that constitute Eiffel Community are self governing, meaning that project related work is done and decisions are taken within the projects.
+
+Governance of the Eiffel Community is the responsibility of Eiffel Technical Committee.
+
+## Projects
+
+The projects that take part in Eiffel Community develop solutions in order to contribute to Eiffel vision of technology agnostic machine-to-machine communication in CI/CD pipelines.
+The work done within the projects can focus on specific areas such as security, visualization, and ease of use or contribute to and drive the high level architecture or the protocol itself which impacts the entire community.
+
+The projects that take part in Eiffel Community currently have two roles relevant to their governance.
+
+### Contributors
+
+A contributor is anyone who made contributions to the projects within Eiffel Community.
+
+There are different means and ways to contribute to Eiffel Community in order to be a contributor, such as issuing and reviewing pull requests, creating issues in project backlog, or taking part in technical discussions.
+Determination of the complete criteria for contributors is the responsibility of Eiffel Technical Committee with the input from Eiffel Community at large.
+
+Contributors are eligible to vote on Technical Committee elections. Contributors do not have the rights to accept pull requests.
+
+Being contributor is a prerequisite for becoming a maintainer to the projects they are contributing to.
+In order for a contributor to become a maintainer, the individual must be nominated by one of the existing maintainers with a reference to the individual's history of contributions to the project.
+The nomination is done as a pull request to the *CODEOWNERS* file in corresponding project's git repository. The approval must be done on the pull request by project's maintainers.
+
+### Maintainers
+
+Maintainers are active contributors and participants in the projects they are maintainers of. Maintainers have the rights to accept pull requests.
+
+Maintainers are eligible to vote on Technical Committee elections. They can be nominated as members to the Technical Committee during Technical Committee elections.
+
+Maintainers have right to nominate contributors to become maintainers and vote on the maintainer promotions for the projects they are maintainers of.
+
+Individuals can be maintainers to multiple projects.
+
+## Technical Committee
+
+The Technical Committee is responsible for the oversight of the Eiffel Open Source Project. Technical Committee is comprised of **X** voting members.
```

Comment:
The whole governance structure is a bit confusing to me also in combination with the naming. Can I assume we are referring to all the projects withing the Eiffel-community here? or what is the Eiffel Open source project? 

Given my assumption perhaps we could rephrase to:
"The Technical Committee is responsible for the oversight of the Eiffel-community open source projects."

...or am I misunderstanding things?

Reply:
your understanding is correct. i think what you suggest, "Eiffel Community Open Source Projects", is a good way to state the responsibility of TC.